modules/npcs/src/main/java/eu/cloudnetservice/modules/npc/platform/bukkit/command/NPCCommand.java

```diff
@@ -580,6 +582,72 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
         // sets the target group of the npc
         case "tg", "targetgroup" -> updatedNpc = NPC.builder(npc).targetGroup(args[2]).build();
 
+        case "r", "rot", "rotate" -> {
+          if (args.length < 4) {
+            sender.sendMessage("§cInvalid usage! Use §6/cn edit rotate <yaw/pitch> <value>§c!");
```

Comment:
Maybe the output should contain the relative indicator? something like: `[~]<yaw/pitch>`? But this might be a bit confusing at first.